pkg/workloadmeta/collectors/internal/kubeapiserver/test_helpers.go

```diff
@@ -0,0 +1,59 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build kubeapiserver && test
+
+package kubeapiserver
+
+import (
+	"context"
+	"testing"
+
+	"github.com/DataDog/datadog-agent/pkg/workloadmeta"
+	"github.com/stretchr/testify/assert"
+	"k8s.io/client-go/kubernetes/fake"
+)
+
+const dummySubscriber = "dummy-subscriber"
+
+func TestFakeHelper(t *testing.T, createResource func(*fake.Clientset) error, newStore storeGenerator, expected []workloadmeta.EventBundle) {
+	// Create a fake client to mock API calls.
+	client := fake.NewSimpleClientset()
+
+	// Creating a fake deployment
+	err := createResource(client)
+	assert.NoError(t, err)
+
+	// Use the fake client in kubeapiserver context.
+	wlm := workloadmeta.NewMockStore()
+	ctx := context.Background()
+	store, _ := newStore(ctx, wlm, client)
+	assert.NoError(t, err)
```

Comment:
This is not needed. `err` has not change since the last `assert.NoError(t, err)`.